### ISSUE TITLE
[Enhancement] "Switch-case" does not warn about single cases in switch

### ISSUE THREAD

**Issue by andreasschroth** [kalessil/phpinspectionsea#1060]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Issue type     | Enhancement                          |
| Plugin         | Php Inspections (EA Extended), 3.0.6.1 |
| Language level | PHP 7.2                        |

Current behaviour:
---

Let's say I have code like this (sure, it's obviously nonsense, but e.g. let's say before there were more cases that got removed):

```php
switch ($strategy) {
    case 'demo':
        $adapter = new DemoAdapter();
        break;
}
```

Right now, no warning from the "Switch-case could be simplified" inspection.

Expected behaviour:
---

I'd expect a warning "Switch construct behaves as if-else, consider refactoring." in the above case. Why? Because basically it's nothing else than a single if.

**Comment by kalessil** [comment]
Confirmed, we have an inspection which supposed to cover this already.

**Comment by kalessil** [comment]
Done!
